pkg/api/types.go

```diff
@@ -2121,6 +2121,16 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
```

Comment:
Can we make the "special" name be something that is out of bounds for real instances?  Maybe "SYSTEM" or "kubernetes.io/system" or "auto:system" or something?  I don't want to debug clusters that have an instance of PriorityClass (where is that part defined, btw - I think it should be this same PR) named "system".